src/filePush/filePush.ts

```diff
@@ -0,0 +1,29 @@
+import Client from 'ssh2-sftp-client';
+import logger from '../util/logger';
+
+
+export const filePush = async (filename: string) => {
+
+  const config = {
+    host: process.env.SFTP_Address,
+    username: process.env.SFTP_User,
+    password: process.env.SFTP_Password,
+    retries: 3, 
+  };
+  
+  const sftp = new Client();
+
+  const remoteFolderLocation = '/upload/';
+  const remoteFileLocation = remoteFolderLocation + filename;
```

Comment:
You could get rid of the line above an just do `const remoteFileLocation = '/upload/' + filename;`,
as we're not using `remoteFolderLocation` anywhere else.